Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent {
+        label 'linux-large'
+        //TODO 6146 windows builds ?using label 'proxmox-windows'?
+    }
+    environment {
+        // What variables do we need here
+        projectName = 'DDF'
```

Comment:
We can pull a little from the pre-exisiting pipelines environment variables. For the nightly(s) we could use the project names and that will likely be descriptive enough.